python 3.6?

Any efforts going on to support python 3.6? Are you accepting PRs?

Fixing Travis seems simple enough; just two flake8 errors.

```
./quamash/_unix.py:220:1: E305 expected 2 blank lines after class or function definition, found 1
./quamash/__init__.py:172:1: E305 expected 2 blank lines after class or function definition, found 1
```
From https://travis-ci.org/harvimt/quamash/jobs/188431070

Is there a big issue holding you back that I'm overlooking? If it's just some linting I'll fix it for you.

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,6 +1,5 @@
 language: python
 python:
-  - "3.3"
   - "3.4"
   - "3.5"
   - "3.6"
@@ -55,8 +54,6 @@ install:
   - popd
   # PySide
   - if [ $TRAVIS_PYTHON_VERSION != '3.5' && $TRAVIS_PYTHON_VERSION != '3.6' ]; then pip install --find-links wheelhouse/ pyside; fi
-  # Python 3.3
-  - if echo $TRAVIS_PYTHON_VERSION | grep -q '^3.3'; then pip install asyncio; fi
   # flake8 style checker
   - pip install flake8 pep8-naming flake8-debugger flake8-docstrings pytest-timeout
 script:

--- a/.travis.yml
+++ b/.travis.yml
@@ -3,6 +3,7 @@ python:
   - "3.3"
   - "3.4"
   - "3.5"
+  - "3.6"
 before_install:
   - export DISPLAY=:99.0
   - sh -e /etc/init.d/xvfb start
@@ -52,7 +53,7 @@ install:
   # Builds Complete
   - popd
   # PySide
-  - if [ $TRAVIS_PYTHON_VERSION != '3.5' ]; then pip install --find-links wheelhouse/ pyside; fi
+  - if [ $TRAVIS_PYTHON_VERSION != '3.5' && $TRAVIS_PYTHON_VERSION != '3.6' ]; then pip install --find-links wheelhouse/ pyside; fi
   # Python 3.3
   - if echo $TRAVIS_PYTHON_VERSION | grep -q '^3.3'; then pip install asyncio; fi
   # flake8 style checker
@@ -60,7 +61,7 @@ install:
 script:
   - flake8
   - flake8 --select=D1 quamash/*.py
-  - if [ $TRAVIS_PYTHON_VERSION != '3.5']; then QUAMASH_QTIMPL=PySide py.test; fi
+  - if [ $TRAVIS_PYTHON_VERSION != '3.5' && $TRAVIS_PYTHON_VERSION != '3.6' ]; then QUAMASH_QTIMPL=PySide py.test; fi
   - QUAMASH_QTIMPL=PyQt4 py.test
   - QUAMASH_QTIMPL=PyQt5 py.test
 cache:

--- a/.travis.yml
+++ b/.travis.yml
@@ -45,6 +45,7 @@ install:
   # PyQt5
   - source /opt/qt55/bin/qt55-env.sh  # switch to Qt5
   - tar xzf /downloads/pyqt5.tar.gz --keep-newer-files
+  - ls -l
   - pushd PyQt-gpl-5.5
   - python configure.py -c --confirm-license --no-designer-plugin -e QtCore -e QtGui -e QtWidgets
   - make

--- a/.travis.yml
+++ b/.travis.yml
@@ -23,9 +23,9 @@ install:
   # Qt4
   - sudo apt-get install -y libqt4-dev
   # Qt5
-  - sudo add-apt-repository -y ppa:beineri/opt-qt542
+  - sudo add-apt-repository -y ppa:beineri/opt-qt551-trusty
   - sudo apt-get update
-  - sudo apt-get install -y qt54base
+  - sudo apt-get install -y qt55base
   # Builds
   - pushd /builds
   # SIP
@@ -43,9 +43,9 @@ install:
   - sudo make install
   - popd
   # PyQt5
-  - source /opt/qt54/bin/qt54-env.sh  # switch to Qt5
+  - source /opt/qt55/bin/qt55-env.sh  # switch to Qt5
   - tar xzf /downloads/pyqt5.tar.gz --keep-newer-files
-  - pushd PyQt-gpl-5.4
+  - pushd PyQt-gpl-5.5
   - python configure.py -c --confirm-license --no-designer-plugin -e QtCore -e QtGui -e QtWidgets
   - make
   - sudo make install

--- a/.travis.yml
+++ b/.travis.yml
@@ -45,7 +45,7 @@ install:
   # PyQt5
   - source /opt/qt59/bin/qt59-env.sh  # switch to Qt5
   - tar xzf /downloads/pyqt5.tar.gz --keep-newer-files
-  - pushd PyQt-gpl-5.9
+  - pushd PyQt5_gpl-5.9
   - python configure.py -c --confirm-license --no-designer-plugin -e QtCore -e QtGui -e QtWidgets
   - make
   - sudo make install

--- a/.travis.yml
+++ b/.travis.yml
@@ -10,17 +10,18 @@ before_install:
   # Cached Downloads
   - sudo mkdir -p /downloads
   - sudo chmod a+rw /downloads
-  - if [ ! -f /downloads/sip.tar.gz ];   then curl -L -o /downloads/sip.tar.gz http://sourceforge.net/projects/pyqt/files/sip/sip-4.16.5/sip-4.16.5.tar.gz; fi
-  - if [ ! -f /downloads/pyqt4.tar.gz ]; then curl -L -o /downloads/pyqt4.tar.gz http://sourceforge.net/projects/pyqt/files/PyQt4/PyQt-4.11.3/PyQt-x11-gpl-4.11.3.tar.gz; fi
+  - if [ ! -f /downloads/sip.tar.gz ];   then curl -L -o /downloads/sip.tar.gz https://sourceforge.net/projects/pyqt/files/sip/sip-4.19.3/sip-4.19.3.tar.gz; fi
+  - if [ ! -f /downloads/pyqt4.tar.gz ]; then curl -L -o /downloads/pyqt4.tar.gz https://sourceforge.net/projects/pyqt/files/PyQt4/PyQt-4.12.1/PyQt4_gpl_x11-4.12.1.tar.gz; fi
   - if [ ! -f /downloads/pyqt5.tar.gz ]; then curl -L -o /downloads/pyqt5.tar.gz https://sourceforge.net/projects/pyqt/files/PyQt5/PyQt-5.9/PyQt5_gpl-5.9.tar.gz; fi
-  - echo '6d01ea966a53e4c7ae5c5e48c40e49e5  /downloads/sip.tar.gz' | md5sum -c -
-  - echo '997c3e443165a89a559e0d96b061bf70  /downloads/pyqt4.tar.gz' | md5sum -c -
+  - echo '4708187f74a4188cb4e294060707106f  /downloads/sip.tar.gz' | md5sum -c -
+  - echo '0112e15858cd7d318a09e7366922f874  /downloads/pyqt4.tar.gz' | md5sum -c -
   - echo 'a409ac0d65ead9178b90c2822759a84b  /downloads/pyqt5.tar.gz' | md5sum -c -
   # Builds
   - sudo mkdir -p /builds
   - sudo chmod a+rw /builds
 install:
   # Qt4
+  - sudo apt-get update
   - sudo apt-get install -y libqt4-dev
   # Qt5
   - sudo add-apt-repository -y ppa:beineri/opt-qt59-trusty
@@ -30,14 +31,14 @@ install:
   - pushd /builds
   # SIP
   - tar xzf /downloads/sip.tar.gz --keep-newer-files
-  - pushd sip-4.16.5
+  - pushd sip-4.19.3
   - python configure.py
   - make
   - sudo make install
   - popd
   # PyQt4
   - tar xzf /downloads/pyqt4.tar.gz --keep-newer-files
-  - pushd PyQt-x11-gpl-4.11.3
+  - pushd PyQt4_gpl_x11-4.12.1
   - python configure.py -c --confirm-license --no-designer-plugin -e QtCore -e QtGui
   - make
   - sudo make install

--- a/.travis.yml
+++ b/.travis.yml
@@ -61,7 +61,7 @@ install:
 script:
   - flake8
   - flake8 --select=D1 quamash/*.py
-  - if [ $TRAVIS_PYTHON_VERSION != '3.5' && $TRAVIS_PYTHON_VERSION != '3.6' ]; then QUAMASH_QTIMPL=PySide py.test; fi
+  - if [ $TRAVIS_PYTHON_VERSION != '3.5' && $TRAVIS_PYTHON_VERSION != '3.6']; then QUAMASH_QTIMPL=PySide py.test; fi
   - QUAMASH_QTIMPL=PyQt4 py.test
   - QUAMASH_QTIMPL=PyQt5 py.test
 cache:

--- a/.travis.yml
+++ b/.travis.yml
@@ -12,10 +12,10 @@ before_install:
   - sudo chmod a+rw /downloads
   - if [ ! -f /downloads/sip.tar.gz ];   then curl -L -o /downloads/sip.tar.gz http://sourceforge.net/projects/pyqt/files/sip/sip-4.16.5/sip-4.16.5.tar.gz; fi
   - if [ ! -f /downloads/pyqt4.tar.gz ]; then curl -L -o /downloads/pyqt4.tar.gz http://sourceforge.net/projects/pyqt/files/PyQt4/PyQt-4.11.3/PyQt-x11-gpl-4.11.3.tar.gz; fi
-  - if [ ! -f /downloads/pyqt5.tar.gz ]; then curl -L -o /downloads/pyqt5.tar.gz http://sourceforge.net/projects/pyqt/files/PyQt5/PyQt-5.4/PyQt-gpl-5.4.tar.gz; fi
+  - if [ ! -f /downloads/pyqt5.tar.gz ]; then curl -L -o /downloads/pyqt5.tar.gz https://sourceforge.net/projects/pyqt/files/PyQt5/PyQt-5.9/PyQt5_gpl-5.9.tar.gz; fi
   - echo '6d01ea966a53e4c7ae5c5e48c40e49e5  /downloads/sip.tar.gz' | md5sum -c -
   - echo '997c3e443165a89a559e0d96b061bf70  /downloads/pyqt4.tar.gz' | md5sum -c -
-  - echo '7f2eb79eaf3d7e5e7df5a4e9c8c9340e  /downloads/pyqt5.tar.gz' | md5sum -c -
+  - echo 'a409ac0d65ead9178b90c2822759a84b  /downloads/pyqt5.tar.gz' | md5sum -c -
   # Builds
   - sudo mkdir -p /builds
   - sudo chmod a+rw /builds
@@ -43,10 +43,9 @@ install:
   - sudo make install
   - popd
   # PyQt5
-  - source /opt/qt55/bin/qt55-env.sh  # switch to Qt5
+  - source /opt/qt59/bin/qt59-env.sh  # switch to Qt5
   - tar xzf /downloads/pyqt5.tar.gz --keep-newer-files
-  - ls -l
-  - pushd PyQt-gpl-5.5
+  - pushd PyQt-gpl-5.9
   - python configure.py -c --confirm-license --no-designer-plugin -e QtCore -e QtGui -e QtWidgets
   - make
   - sudo make install

--- a/.travis.yml
+++ b/.travis.yml
@@ -23,9 +23,9 @@ install:
   # Qt4
   - sudo apt-get install -y libqt4-dev
   # Qt5
-  - sudo add-apt-repository -y ppa:beineri/opt-qt551-trusty
+  - sudo add-apt-repository -y ppa:beineri/opt-qt59-trusty
   - sudo apt-get update
-  - sudo apt-get install -y qt55base
+  - sudo apt-get install -y qt59base
   # Builds
   - pushd /builds
   # SIP

--- a/appveyor.yml
+++ b/appveyor.yml
@@ -21,22 +21,22 @@ environment:
       PYTHON_ARCH: "32"
       QTIMPL: "PySide"
     
-    - PYTHON: "C:\Python35"
+    - PYTHON: "C:\\Python35"
       PYTHON_VERSION: "3.5.3"
       PYTHON_ARCH: "32"
       QTIMPL: "PyQt4"
 
-    - PYTHON: "C:\Python35"
+    - PYTHON: "C:\\Python35"
       PYTHON_VERSION: "3.5.3"
       PYTHON_ARCH: "32"
       QTIMPL: "PyQt5"
       
-    - PYTHON: "C:\Python36"
+    - PYTHON: "C:\\Python36"
       PYTHON_VERSION: "3.6.1"
       PYTHON_ARCH: "32"
       QTIMPL: "PyQt4"
 
-    - PYTHON: "C:\Python36"
+    - PYTHON: "C:\\Python36"
       PYTHON_VERSION: "3.6.1"
       PYTHON_ARCH: "32"
       QTIMPL: "PyQt5"

--- a/appveyor.yml
+++ b/appveyor.yml
@@ -20,6 +20,26 @@ environment:
       PYTHON_VERSION: "3.3.5"
       PYTHON_ARCH: "32"
       QTIMPL: "PySide"
+    
+    - PYTHON: "C:\Python35"
+      PYTHON_VERSION: "3.5.3"
+      PYTHON_ARCH: "32"
+      QTIMPL: "PyQt4"
+
+    - PYTHON: "C:\Python35"
+      PYTHON_VERSION: "3.5.3"
+      PYTHON_ARCH: "32"
+      QTIMPL: "PyQt5"
+      
+    - PYTHON: "C:\Python36"
+      PYTHON_VERSION: "3.6.1"
+      PYTHON_ARCH: "32"
+      QTIMPL: "PyQt4"
+
+    - PYTHON: "C:\Python36"
+      PYTHON_VERSION: "3.6.1"
+      PYTHON_ARCH: "32"
+      QTIMPL: "PyQt5"
 
 init:
   - "ECHO %PYTHON% %PYTHON_VERSION% %PYTHON_ARCH% %QTIMPL%"

